### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by bording**
Given that we are going ahead and releasing a version of the host that works with NSB 7, here is a PR that updates everything to use the new project system and builds the packages using Particular.Packaging.

This does result in a packages that have a different description than before, because the old description was outdated.

I instead kept the assembly's description, which now gets used as the package description.

### CODE REVIEW

File: src/NServiceBus.Host32/NServiceBus.Host32.csproj [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    <PackageReference Include="Fody" Version="2.2.0" PrivateAssets="All" />
+    <PackageReference Include="log4net" Version="1.2.10" PrivateAssets="All" />
+    <PackageReference Include="NServiceBus" Version="[7.0.0-beta0011, 8.0.0)" />
+    <PackageReference Include="Obsolete.Fody" Version="4.3.2" PrivateAssets="All" />
+    <PackageReference Include="Particular.CodeRules" Version="0.1.1" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="..\NServiceBus.Hosting.Windows\**\*.cs" Exclude="..\NServiceBus.Hosting.Windows\obj\**\*.cs" />
```
Comment (ramonsmits) [inline src/NServiceBus.Host32/NServiceBus.Host32.csproj]: What is up with this exclusion? Why is this needed? Obj would be excluded by default if I'm not mistaken?

Comment (bording) [reply src/NServiceBus.Host32/NServiceBus.Host32.csproj]: This because it's adding all the files from the other folder, and no, obj is not excluded by default.
